odoo/addons/test_lint/tests/test_i18n.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import re
+
+from . import lint_case
+
+from odoo import tools
+
+_logger = logging.getLogger(__name__)
+
+
+class TestI18n(lint_case.LintCase):
+    # Assumption: Text starting with an uppercase letter is likely supposed to be translated.
+    PROPS_RE = re.compile(
+    r"""
+        (
+            <[A-Z][^>]+\s+                  # Match a component opening tag with a capital letter, followed by attributes
+            (
+                [a-zA-Z-]+="[^"]*"\s+       # Match attribute name="value" patterns
+            )*                              # Zero or more such attributes
+        )
+        (?!t-)                              # Negative lookahead to exclude attributes starting with t-
```

Comment:
lookahead: to match something not followed by something else
lookbehind: to match something not preceded by something else

In this case, you want to discard attributes not preceded by `t-`, so I guess it's a negative lookbehind, not a lookahead?

rewording suggestion: exclude directives (attributes starting with t-)

Reply:
Oh! you are right. Thanks :))